token/token.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 
 	"github.com/dgrijalva/jwt-go"
-	authclient "github.com/fabric8-services/fabric8-auth/token"
+	authjwk "github.com/fabric8-services/fabric8-auth/token/jwk"
```

Comment:
never heard of jwk only jwt.

Reply:
`jwk` is for `JSON Web Keys`, as @alexeykazakov suggested in https://github.com/fabric8-services/fabric8-auth/pull/434#pullrequestreview-111325525